### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by IhorPokr**
- [DEMO LINK](https://IhorPokr.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://IhorPokr.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

**Comment by alexandra-protyanova** [comment]
Снимок экрана 2022-10-21 в 10 10 36
let us remove default blue outline from input

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="margin-form"
+    action="#"
+    method="post"
+    data-qa="big"
+    >
```
Comment (alexandra-protyanova) [inline src/index.html:19]:
```suggestion
    <form
      class="margin-form"
      action="#"
      method="post"
      data-qa="big"
    >
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.margin-form {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.input {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```
Comment (alexandra-protyanova) [inline src/style.css:18]: move to `html` selector

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.margin-form {
+  margin-top: 20px;
```
Comment (alexandra-protyanova) [inline src/style.css:13]: combine margins

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+.input {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  width: 100%;
+  height: 70px;
+  line-height: 22px;
+  padding-left: 62px;
+  box-sizing: border-box;
```
Comment (alexandra-protyanova) [inline src/style.css:25]: move to `*` selector

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+.input:focus {
+  font-weight: bold;
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: absolute;
+  background-position-x: 26px;
+  background-position-y: 25px;
+  background-size: 19px 19px;
```
Comment (alexandra-protyanova) [inline src/style.css:51]: u r repeating your background for 4 times, it is not ok. Use tag `img` or pseudoelement for background image

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 3 above)
Comment (IhorPokr) [reply src/style.css:13]: wym by combine margins?

Comment (alexandra-protyanova) [reply src/style.css:21]: u have top & bottom margins, set it as `margin: 20px 0;`

File: src/style.css (hunk 5 above)
Comment (alexandra-protyanova) [reply src/style.css:57]: haven't fixed

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]
Fix all previous comments

**Pull request closed without merging**